charts/configs/bitcoind/localdev.yaml

```diff
@@ -0,0 +1,2 @@
+service:
```

Comment:
maybe add a comment why we need that (necessary for minikube to be able to access from outside the cluster?)